onadata/libs/constants.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals, absolute_import
+
+# Userprofile Permissions
+CAN_ADD_USERPROFILE = 'add_userprofile'
+CAN_CHANGE_USERPROFILE = 'change_userprofile'
+CAN_DELETE_USERPROFILE = 'delete_userprofile'
+CAN_ADD_XFORM_TO_PROFILE = 'can_add_xform'
+CAN_VIEW_PROFILE = 'view_profile'
+
+# Organization Permissions
+CAN_VIEW_ORGANIZATION_PROFILE = 'view_organizationprofile'
+CAN_ADD_ORGANIZATION_PROFILE = 'add_organizationprofile'
+CAN_ADD_ORGANIZATION_XFORM = 'can_add_xform'
+CAN_CHANGE_ORGANIZATION_PROFILE = 'change_organizationprofile'
+CAN_DELETE_ORGANIZATION_PROFILE = 'delete_organizationprofile'
+IS_ORGANIZATION_OWNER = 'is_org_owner'
+
+# Xform Permissions
+CAN_CHANGE_XFORM = 'change_xform'
```

Comment:
Indeed but I would have to change many parts of the code we are going to remove soon. 
To avoid a ton of conflicts, these changes could be done `kobokitten-remove-ui-CUD-actions-unicode` instead.
What do you think?

Reply:
Yes, let's not create merge conflicts.